pkg/model/schema.go

```diff
@@ -39,8 +39,8 @@ func mergeAndSortUnique(slice1, slice2 []string) []string {
 	return result
 }
 
-// getStructKeys recursively goes through a given struct and returns a map of struct names to their fields
-func getStructKeys(t interface{}) map[string][]string {
+// GetStructKeys recursively goes through a given struct and returns a map of struct names to their fields
+func GetStructKeys(t interface{}) map[string][]string {
```

Comment:
As we created a ´schema´ pkg, i believe this one now belongs to the schema pkg? i see the logic of extracting the keys as part of it... also might move the manifest friendy error logic too and exported from the schema pkg... wdyt?

Reply:
@teresaromero I tried but as they are very related, it creates a cycle import between `pkg/model` and `pkg/schema`